### DISCUSSION TITLE
[RFC 001] Model Provider Support Proposals (模型供應商支持請求)

### DISCUSSION THREAD

**Discussion by zmh-program** [coaidev/coai#277] [Polls | 投票]
In order to keep up with the new model development, we are centralizing this issue to document your needs and priorities for new model providers

New provider submission format:
**Provider name + API documentation + introduction**

We will prioritize the support based on the number of upvotes in the emoticons below the comments.

**Comment by zmh-program** [comment]
NovelAI / novelai.net / Image Generation

**Comment by zmh-program** [comment]
Baidu ERNIE / https://cloud.baidu.com/doc/WENXINWORKSHOP/s/flfmc9do2

**Comment by zmh-program** [comment]
AWS Bedrock / https://aws.amazon.com/cn/bedrock

**Comment by zmh-program** [comment]
Google Vertex AI / https://cloud.google.com/vertex-ai

**Comment by zmh-program** [comment]
Cloudflare AI / https://ai.cloudflare.com

**Comment by zmh-program** [comment]
01.AI / https://01.ai

**Comment by zmh-program** [comment]
Cohere (Command R) / https://cohere.com/command

**Comment by zmh-program** [comment]
OpenRouter / https://openrouter.ai

**Comment by zmh-program** [comment]
Minimax / https://www.minimaxi.com

**Comment by zmh-program** [comment]
Step / https://www.stepfun.com

**Comment by xiaoheiCat** [comment]
Coze 扣子 / https://www.coze.cn/docs/developer_guides/chat_v3 & https://www.coze.com/docs/developer_guides/chat_v3，/ Coze 扣子是字节旗下的 AI Bot 开发平台，目前支持部分用户将自己制作的 Bot 发表为 API 进行调用。注意：Coze 扣子分为国版 (coze.cn) 与国际版 (coze.com)，相互之间账号与制作的 Bot 不通用，接入时请让用户确认 API 域名。

**Reply by Sh1n3zZ** [reply]
supported

[a comment by an automated account is not shown]

**Comment by mercury7720** [comment]
The new API endpoint of Tencent Hunyuan Model is `hunyuan.tencentcloudapi.com`. Tencent Cloud customer service said that the current connected version will be offline in August.

hunyuan-pro... / https://cloud.tencent.com/document/product/1729/105701

**Comment by 360troubleWind** [comment]
dify的应用中的API，文档将会在会在dify用户的应用内部展现
![image](https://github.com/user-attachments/assets/3f043a5b-2215-4ba7-90c9-caf187de6711)
dify官方网址：[dify](https://dify.ai/)

**Reply by Sh1n3zZ** [reply]
supported

[a comment by an automated account is not shown]

**Comment by acleverfreebird** [comment]
支持智谱清影制作视频大模型
[API文档](https://maas.aminer.cn/dev/api#cogvideox)
模型名CogVideoX

[a comment by an automated account is not shown]

**Comment by zydmtaichi** [comment]
Stable Diffusion web UI / [API文档](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/API)
A web interface for Stable Diffusion, implemented using Gradio library.

**Comment by mosesyyoung** [comment]
llama3.1/3.2, managed with newest ollama.

[a comment by an automated account is not shown]

**Comment by Shulychee** [comment]
siliconflow [API DOC](https://docs.siliconflow.cn/api-reference/chat-completions/chat-completions) + Based on an excellent open-source foundational model, we provide cost-effective GenAI services, similar to Groq Service.

**Reply by kitaev-chen** [reply]
挂一个图吸引注意力：
![image](https://github.com/user-attachments/assets/d49a61e7-c391-4753-8634-e1f475da84b3)

**Reply by Synex93** [reply]
> 挂一个图吸引注意力： ![image](https://private-user-images.githubusercontent.com/6076946/392555066-d49a61e7-c391-4753-8634-e1f475da84b3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk2OTA5MTEsIm5iZiI6MTczOTY5MDYxMSwicGF0aCI6Ii82MDc2OTQ2LzM5MjU1NTA2Ni1kNDlhNjFlNy1jMzkxLTQ3NTMtODYzNC1lMWY0NzVkYTg0YjMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTAyMTZUMDcyMzMxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTg0OTFiYTIzYjY1MzU2NzQ2NmE2ZDEwMjhjNDkwNTBlYWNlMGZiZGUzZmNiYTYxY2NjZDljMDQwNjVhYTQ3MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.NegNzURNUnth3N3VV2SJB5S6MNdlyLEDnTb7Q2WxnlU)

这个是不是还没支持。。

**Comment by jasonhp** [comment]
[Novita AI](https://novita.ai/model-api/product/llm-api)
Novita is the go-to inference platform for AI developers seeking a low-cost, reliable, and simple solution for shipping AI models.

**Comment by zmh-program** [comment]
xAI / console.x.ai

**Reply by kitaev-chen** [reply]
这个不错，每月都有25刀免费额度。

**Comment by Synex93** [comment]
siliconflow [API DOC](https://docs.siliconflow.cn/api-reference/chat-completions/chat-completions) + Based on an excellent open-source foundational model, we provide cost-effective GenAI services, similar to Groq Service.

**Comment by ati9527** [comment]
火山引擎+[接入文档](https://www.volcengine.com/docs/82379/1494384?redirect=1) 火山默认后面是个V3而不是v1，导致无法通过openai兼容模式调用
